src/rust/python-parser/cxx-bridge.c++

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+// #include <workerd/rust/python-parser/lib.rs.h>
+#include "workerd/rust/cxx-integration/lib.rs.h"
+
+#include <kj/test.h>
+
+namespace workerd {
+namespace rust {
+namespace python_parser {
+  ::rust::Vec<::rust::String> get_imports(::rust::Slice<::rust::Str const> sources);
```

Comment:
I don't love declaring this manually, but at least C++ name mangling means there is a very high probability of a link error rather than generating a bad executable if the declaration is wrong. The problem is this is declared in `workerd/rust/python-parser/lib.rs.h` which is generated by `wd_rust_crate` but that is a dependent of the rule that compiles `cxx-bridge.c++`. This probably means I'm doing something wrong in Bazel? cc @mikea

Reply:
Try using the cxx_bridge_src bazel parameter to set this up, like in src/rust/cxx-integration/.